datadog-integrations-aws-handler/requirements.txt

```diff
@@ -1 +1 @@
-git+https://github.com/datadog/datadog-cloudformation-resources.git@datadog-cloudformation-common-python-0.0.17#egg=datadog_cloudformation_common&subdirectory=datadog-cloudformation-common-python
+git+https://github.com/datadog/datadog-cloudformation-resources.git@0b54e768b81bf33e75e774b14fb938698b7add27#egg=datadog_cloudformation_common&subdirectory=datadog-cloudformation-common-python
```

Comment:
This is referencing a particular commit. This will be changed to reference a release before it goes out. I can't reference the release as it's not out yet.

Reply:
updated after merging api version bump https://github.com/DataDog/datadog-cloudformation-resources/pull/316